srcpkgs/lhasa/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'lhasa'
+pkgname=lhasa
+version=0.3.1
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake libtool"
+depends="liblhasa>=${version}_${revision}"
+checkdepends="tzdata"
```

Comment:
We added `tzdata` to the base system, this just needs to be propagated to CI. Even removing it from here, you should be able to run `./xbps-src -Q pkg lhasa` locally with success.

Reply:
I tried that after git pull and rebasing to master but I got same error as the original commit.

Reply:
To be fixed by #31641

Reply:
You need a `./xbps-src bootstrap-update`